src/lib/operators/join_hash/join_hash_steps.hpp

```diff
@@ -596,20 +619,20 @@ void probe_semi_anti(const RadixContainer<RightType>& radix_container,
 }
 
 using PosLists = std::vector<std::shared_ptr<const PosList>>;
-using PosListsBySegment = std::vector<std::shared_ptr<PosLists>>;
+using PosListsByChunk = std::vector<std::shared_ptr<PosLists>>;
 
 /**
  * Returns a vector where each entry with index i references a PosLists object. The PosLists object
  * contains the position list of every segment/chunk in column i.
  * @param input_table
  */
 // See usage in _on_execute() for doc.
-inline PosListsBySegment setup_pos_lists_by_segment(const std::shared_ptr<const Table>& input_table) {
+inline PosListsByChunk setup_pos_lists_by_segment(const std::shared_ptr<const Table>& input_table) {
```

Comment:
Return type and function name are confusing. Is it now by chunk or segment?

Reply:
Yep, changed return type name without changing the function name. Fixed